app/controllers/concerns/staff_response_context.rb

```diff
@@ -47,8 +48,9 @@ def visit_params
         :nomis_id,
         :banned,
         :not_on_list,
-        banned_until: %i[day month year]
-      ]
+        banned_until: [:day, :month, :year]
```

Comment:
Interesting, I think this must have been a rubocop autocorrection as I didn't change it. I'll revert and re-run rubocop to double check